docs/console/project-configuration/manage-environment-variables/manage-environment-variables-with-akv.mdx

```diff
@@ -0,0 +1,438 @@
+
+The first step in order to let your AKV integration work is to install the [External Secret Operator](https://external-secrets.io/latest/).
+
+In this example we will use [_Helm_](https://helm.sh/) to install it, but you can find the complete installation guide at [this page](https://external-secrets.io/latest/introduction/getting-started/).
+
+:::warning
+Before installing the External Secret Operator, be sure to **be connected to the right k8s cluster**.
+:::
+
+With the following simple commands you can install the External Secret Operator:
+
+```bash
+helm repo add external-secrets https://charts.external-secrets.io
+
+helm install external-secrets \
+   external-secrets/external-secrets \
+    -n external-secrets \
+    --create-namespace
+```
+
+The commands above will install the External Secret Operator in a dedicated namespace, automatically created by _Helm_ during the installation process.
+After the installation is complete, you should have 3 services up and running inside the `external-secrets` namespace:
+
+```bash
+$ kubectl get pods
+NAME                                                         READY   STATUS    RESTARTS   AGE
+external-secrets-operator-5db95c68b8-nn27p                   1/1     Running   0          6d
+external-secrets-operator-cert-controller-568bd5d5cc-258j2   1/1     Running   0          6d
+external-secrets-operator-webhook-85695d9f88-7mxbp           1/1     Running   0          6d
+```
+
+### 2. Azure setup
+
+Azure needs different operations to let you complete the integration with the Mia-Platform Console and the External Secret Operator:
+1. Create an Azure App Registration (aka _Service Principal_);
+2. Create the _Service Principal_ credentials or certificate, to allow the login using it;
+3. Add the "Federated Credentials" to the _Service Principal_, in order to let AKV authenticate with it through the k8s Cluster;
+4. Authorize the _Service Principal_ reading data from the Azure Key Vault you need to access.
+
+:::info
+In this example we used the _Azure Portal_ to perform the operations above. If you prefer you can use the `az` client or the Azure REST APIs too.
+:::
+
+#### 1. Create the Service Principal
+
+To create the Service Principal you just need to:
+* open the Azure Portal _App Registrations_ page on your browser (usually the page is at [this link](https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade), but it can be different based on your Azure subscription);
+* click on the "New registration" button;
+* insert the name you prefer;
+* select the account types &rarr; usually the "Accounts in this organizational directory only" is the used option;
+* leave the "Redirect URI" empty;
+* click on the "Register" button.
+
+![azurePortal-app-registration-creation](../img/azurePortal-app-registration-creation.png)
+
+:::info
+**Save the `clientId` generated by Azure**, the `Application (client) ID` field content, because you will need it later.
+:::
```

Comment:
These extremely detailed instructions could get outdated very soon, have we considered providing `az` client shell commands instead? I expect a more stable interface from that.

Reply:
You are right, I'll add an `az` client commands section.

Reply:
@mattia-fumo done

Reply:
Hiiii, is it ok now?

Reply:
@mattia-fumo can you do a last review so that we can merge this? thanks!

Reply:
@mattia-fumo @fredmaggiowski still waiting 😢

Reply:
LGTM